deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -472,6 +472,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                         isSelected: o.id === currentCluster.baseImageId
                     }
                 }),
+                useLaunchTemplate: currentCluster.useLaunchTemplate != null ? currentCluster.useLaunchTemplate : false,
```

Comment:
What is the case when currentCluster.useLaunchTemplate == null?

Reply:
When we roll back the rodimus side to current master branch which doesnt have the useLaunchTemplate code, we want to default to false.